### ISSUE TITLE
Reference to 'remote pull'

### ISSUE THREAD

**Issue by tdmalone** [hashicorp/terraform-website#238]
I'm new to Terraform, but I'm guessing this refers to an old command - as it doesn't appear to be a valid command anymore:

> After configuring remote state, run a `terraform remote pull` command to sync state locally. The `terraform console` command will use this state for operations.

(from https://www.terraform.io/docs/commands/console.html#remote-state)

**Comment by saurabh-hirani** [comment]
I also had the same observation. Was about to open an issue and came across this one. The command should be "terraform state pull"

**Comment by nfagerlund** [comment]
Martin fixed this at some point during the 0.12 run-up.

**Comment by chaddjohnson** [comment]
`terraform remote pull` seems to be a non-existent command, yet it is still listed in the documentation: https://www.terraform.io/docs/commands/state/pull.html

```
hyperion:infrastructure chad$ terraform remote pull
Usage: terraform [-version] [-help] <command> [args]

The available commands for execution are listed below.
The most common, useful commands are shown first, followed by
less common or more advanced commands. If you're just getting
started with Terraform, stick with the common commands. For the
other commands, please read the help and docs before usage.

Common commands:
    apply              Builds or changes infrastructure
    console            Interactive console for Terraform interpolations
    destroy            Destroy Terraform-managed infrastructure
    env                Workspace management
    fmt                Rewrites config files to canonical format
    get                Download and install modules for the configuration
    graph              Create a visual graph of Terraform resources
    import             Import existing infrastructure into Terraform
    init               Initialize a Terraform working directory
    output             Read an output from a state file
    plan               Generate and show an execution plan
    providers          Prints a tree of the providers used in the configuration
    refresh            Update local state file against real resources
    show               Inspect Terraform state or plan
    taint              Manually mark a resource for recreation
    untaint            Manually unmark a resource as tainted
    validate           Validates the Terraform files
    version            Prints the Terraform version
    workspace          Workspace management

All other commands:
    0.12upgrade        Rewrites pre-0.12 module source code for v0.12
    debug              Debug output management (experimental)
    force-unlock       Manually unlock the terraform state
    push               Obsolete command for Terraform Enterprise legacy (v1)
    state              Advanced state management
```

Is this command no longer available?
